packages/remirror__messages/scripts/messages-to-json.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-env node */
```

Comment:
Could you explain a bit why we need all these JSON files? I must miss something.

Reply:
I know. These JSON files are part of the public API in package.json

Reply:
This is so other i18n libraries can consume the messages, without having to load `@remirror/i18n` and hence Lingui